test/ch/type_test.exs

```diff
@@ -0,0 +1,304 @@
+defmodule Ch.TypeTest do
+  use Ecto.Integration.Case
+
+  import Ecto.Query
+  import Bitwise
+
+  alias Ecto.Integration.TestRepo
+
+  describe "field options" do
+    defmodule Defaults do
+      use Ecto.Schema
+
+      @primary_key false
+      schema "ch_defaults" do
+        field :i8, Ch, type: "Int8", default: 42
+        field :string, Ch, type: "String", default: "42"
```

Comment:
this doesn't work, find out why and open a PR into Ecto to support `field` options in parameterised Ecto types